### PR TITLE
Merge rbd cloning edited

### PR DESCRIPTION
**Pull request by alop**
Using this code in alln integration, I was able to get efficient RBD cloning of ephemeral instances without the need to download the image from ceph, and then upload it to glance

Rally-bug: TA8264
Rally-bug: DE1186
Not-in-upstream: true

### CODE REVIEW

**Comment by alop** [comment]
This pull request is 4 commits smaller. When I investigated the 4 that were not included here, I find that they were already cherry-picked.

**Comment by alop** [comment]
Hang on a sec with this one, I'm verifying that it was a clean merge.

**Comment by markvoelker** [comment]
Thanks, can you also propagate "Closes-rally-bug: DE1186" and "Implements: rally-task: TA8264" tags?

File: nova/compute/manager.py [hunk 1]
```diff
@@ -668,10 +668,9 @@ def _init_instance(self, context, instance):
                 return
 
         net_info = compute_utils.get_nw_info_for_instance(instance)
-        try:
```
Comment (alop) [inline nova/compute/manager.py:671]: This is just wrong

**Comment by retr0h** [comment]
This is the messed up PR.  Closing it.

**Pull request closed without merging**